modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -64,6 +64,11 @@ CREATE OR REPLACE DATABASE name
 [[manage-databases-create-database-options]]
 === Options
 
+[.tabbed-example]
+=====
+[.include-with-cypher5]
```

Comment:
```suggestion
[.include-with-cypher-5]
```

If this change is applied to this and other `include-with...` lines, it will update the generated tab title from eg `Cypher5` and `Cypher25` to `Cypher 5` etc (space between `Cypher` and `5`).